java/ql/lib/change-notes/2022-08-09-android-manifest-new-class-and-predicates.md

```diff
@@ -0,0 +1,8 @@
+---
+category: feature
+---
+* Added a new predicate, `requiresPermissions`, in the `AndroidComponentXmlElement` and `AndroidApplicationXmlElement` classes to detect if the element has explicitly set a value for its `android:permission` attribute. Also added `override` annotation to the pre-existing `requiresPermissions` predicate in the `AndroidProviderXmlElement` class.
```

Comment:
```suggestion
* Added a new predicate, `requiresPermissions`, in the `AndroidComponentXmlElement` and `AndroidApplicationXmlElement` classes to detect if the element has explicitly set a value for its `android:permission` attribute.
```

No need to include this level of detail in the change notes.

Reply:
Fixed in commit [cf88d01](https://github.com/github/codeql/pull/9983/commits/cf88d01af67e0e3a4d0f66b699178d39c4846bf6).